packages/client/lib/client/socket.ts

```diff
@@ -82,6 +82,15 @@ export default class RedisSocket extends EventEmitter {
   }
 
   #createReconnectStrategy(options?: RedisSocketOptions): ReconnectStrategyFunction {
+    const defaultStrategy = (retries: number) => {
```

Comment:
I think this should be moved to a `#defaultReconnectStrategy` function and used from lines 109 and 114

Reply:
Done in e7304ada8 👍